chart/templates/worker-deployment.yaml

```diff
@@ -105,14 +129,46 @@ spec:
         {{- if $root.Values.clamav.enabled  }}
         - name: clamav
           image: "{{ $root.Values.clamav.image.value }}:{{ $root.Values.clamav.image.tag }}"
-          imagePullPolicy: {{ $root.Values.image.pullPolicy }}
+          imagePullPolicy: {{ $root.Values.clamav.image.pullPolicy }}
           ports:
-{{ toYaml $root.Values.clamav.ports | indent 12 }}
-{{ toYaml $root.Values.clamav.probes | indent 10 }}
+            - containerPort: 3310
+              name: clamavport
+              protocol: TCP
+          readinessProbe:
```

Comment:
Probe values could also be abstracted into the values.yaml

Reply:
I disagree - the probe should be the same in all environments. One of the environment should not have a different port or protocol for probes that the other environments. Timeouts, and wait times maybe, but definitely not internal routing. Please see the readiness probe where their are environmental differences, and the values were sourced from the values,yaml file of the parent helm chart (and therefore can be over-written by the environments values.yaml file as needed)